layouts/partials/get-started.html

```diff
@@ -7,33 +7,22 @@ <h2>Get Started with Pulumi</h2>
                     Pulumi works with your favorite language, and can be used with any cloud.
                 </p>
             </div>
-            <div class="md:w-3/12 align-middle">
-                <div class="flex align-middle items-center">
-                    <h5 class="flex items-center whitespace-no-wrap">
-                        <span class="border-2 border-blue-500 rounded-full bg-white text-blue-600 w-8 h-8 text-center mr-2 text-green-500">1</span>
-                        Install Pulumi
-                    </h5>
-                </div>
-                <ul class="list-inside list-none pl-8 ml-2">
-                    <li class="leading-snug"><a class="text-blue-700 text-sm" href="{{ relref . "/docs/reference/install" }}">Setup</a></li>
-                </ul>
-            </div>
```

Comment:
Cleaned this up while making changes here (to remove the `/docs/tour` link). We don't really want to send folks to the install page first, we want them to go to the Get Started flow. Originally I kept it 3 items: Get Started, Try Our Tutorials, and Read the Docs, but then the first two steps only had a single link and it didn't look quite right. So collapsed to 2 items.